swarmrl/models/ml_model.py

```diff
@@ -96,15 +108,13 @@ def calc_action(
                 if str(colloid.type) == item:
                     actions[colloid.id] = chosen_actions[count]
                     count += 1
-        actions = list(actions.values())  # convert to a list.
+
         # Record the trajectory if required.
-        if self.record_traj:
-            for item in self.particle_types:
-                record_trajectory(
-                    particle_type=item,
-                    features=np.array(observables[item]),
-                    actions=np.array(action_indices[item]),
-                    log_probs=np.array(log_probs[item]),
-                    rewards=np.array(rewards[item]),
-                )
-        return actions
+        # if self.record_traj:
+        for item in self.particle_types:
```

Comment:
I've noticed that using `type_` as iteration variable instead of `item` makes the code easier to understand for me. 
How do you feel about that?